include/cuco/detail/static_map.inl

```diff
@@ -716,9 +716,11 @@ static_map<Key, Value, Scope, Allocator>::device_view::find(CG g,
 }
 
 template <typename Key, typename Value, cuda::thread_scope Scope, typename Allocator>
-template <typename Hash, typename KeyEqual>
-__device__ bool static_map<Key, Value, Scope, Allocator>::device_view::contains(
-  Key const& k, Hash hash, KeyEqual key_equal) const noexcept
+template <typename ProbeKey, typename Hash, typename KeyEqual>
+__device__ std::enable_if_t<std::is_convertible_v<ProbeKey, Key>, bool>
+static_map<Key, Value, Scope, Allocator>::device_view::contains(ProbeKey const& k,
```

Comment:
Changing type of `k` may not be enough, as I see that `initial_slot` relies on `Key` type.